src/Writer/Map.php

```diff
@@ -91,6 +86,11 @@ public function write($map, int $indentSize = 0, string $indent = self::WRITER_I
             $this->text .= (new Web())->write($map->web, $indentSize + 1, $indent);
         }
 
+        foreach ($map->outputformat as $outputformat) {
+            $this->text .= PHP_EOL;
+            $this->text .= (new OutputFormat())->write($outputformat, $indentSize + 1, $indent);
+        }
+
```

Comment:
I try to sort the parameters alphabetically for consistency.
So logic is good but it should go where it was (between LEGEND and PROJECTION).

Reply:
It's ok, I fixed it.